src/main/java/com/ceos19/springboot/service/MessageService.java

```diff
@@ -0,0 +1,26 @@
+package com.ceos19.springboot.service;
+
+import com.ceos19.springboot.domain.Message;
+import com.ceos19.springboot.domain.Users;
+import com.ceos19.springboot.repository.MessageRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class MessageService {
+
+    private final MessageRepository messageRepository;
```

Comment:
```suggestion
    private final MessageRepository messageRepository;
    
```

전반적으로 코드 포맷팅이 일관적이지 않은 부분들이 있는 것 같아요~ 
지금은 개인 프로젝트이니 크게 상관 없겠지만 협업 시에는 코드 포맷팅도 고려해주시는게 좋습니다!